### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by BranBayou**
### In this project I did the following.

- Implemented view design based on the sneak peek wireframes.

-    view for user
-    view for post

- Analyzed the wireframes deeply and identified all the necessary elements and how a user would navigate from one page to another.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>This is index page of users</h1>
+<ul class='user-list'>
+<% @users.each do |user|%>
+<li>
+  <%= render 'shared/user', user: user%>
+</li>
+<%end%>
+</ul>
```
Comment (codepantha) [inline app/views/users/index.html.erb:7]: - [ ] Please ensure that the index page implements the design from the wireframe. You don't have to style it but kindly ensure that the wireframe design is implemented 👍 
### Wireframe design
image

### Your implementation
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,13 @@
...
+<% if @user.recent_posts.empty? %>
+  <p class='post-card-no-click'>No posts</p>
+<% else %>
+<%@user.recent_posts.each do |post|%>
+<li>
+  <%=render 'shared/post', post: post%>
+</li>
+<%end%>
+<%end%>
+<div class='see-all-post-btn-container'><%= link_to "See all posts", user_posts_path(@user), class: 'see-all-post-btn' %></div>
```
Comment (codepantha) [inline app/views/users/show.html.erb:13]: - [ ] Also kindly ensure that the user show page follows and closely implements the wireframe guidelines as shown below 👇 
### Wireframe
image

### Your Implementation
image

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,14 @@
...
+  <%else%>
+  <%@user.posts.each do |post|%>
+    <li>
+      <%=render 'shared/post', post: post%>
+      <%= render 'shared/comment', comments: post.recent_comments%>
+    </li>
+  <%end%>
+  <%end%>
+</ul>
+<div class='see-all-post-btn-container'><a class='see-all-post-btn'>Pagination</a></div>
```
Comment (codepantha) [inline app/views/posts/index.html.erb]: - [ ] An error occurs while trying to show all posts that belong to a user. Kindly see the error below and try to fix it 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @BranBayou 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Good commit history
✔️ No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @BranBayou, 

## STATUS APPROVED ✔️ 🎉 🎉 ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] Requested changes are made ✔️ 
- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**